lib/api_entity.rb

```diff
@@ -45,7 +45,7 @@ def url
     delegate :relationships, to: :class
 
     def inspect
-      keys_to_exclude = relationships + [:casted_by]
+      keys_to_exclude = relationships.to_a + [:casted_by]
```

Comment:
I'm curious about the need for this

Reply:
to prevent this error:
`this is to fix NoMethodError: undefined method `+' for nil:NilClass`